WordPress/Classes/Services/EditorSettingsService.swift

```diff
@@ -35,11 +49,20 @@ import Foundation
         blog.webEditor = settings.web.rawValue
     }
 
+    private func migrateLocalSettingToRemote(for blog: Blog, success: @escaping () -> Void, failure: @escaping (Swift.Error) -> Void) {
+        if blog.editor.mobile == nil {
+            let settings = GutenbergSettings()
+            let defaultEditor = settings.getDefaultEditor(for: blog)
+            settings.setGutenbergEnabled(defaultEditor == .gutenberg, for: blog)
```

Comment:
I don't completely follow this part. It seems that if there is no mobile setting stored we would store what we decide is the default one (Gutenberg for .com/Jetpack, Aztec for self hosted). However we always set a value for this in the migration, so it should never be nil?

I'm wondering if we should make `Blog.mobileEditor` non-optional in the data model, and ensure it has the `getDefaultEditor` value when a blog instance is created.

Reply:
> However we always set a value for this in the migration, so it should never be nil?

It will be nil on new installs, right? 🤔 

> I'm wondering if we should make Blog.mobileEditor non-optional in the data model, and ensure it has the getDefaultEditor value when a blog instance is created.

Sounds like a good alternative. I don't think I will have time to check it out today though